spatial-index-h3/.env

```diff
@@ -0,0 +1,7 @@
+# API Base URL (copy this from CARTO Workspace -> Developers)
```

Comment:
The name of the folder should be: spatial-features-h3

Reply:
Renamed